.github/workflows/pr.yaml

```
@@ -662,14 +661,13 @@ jobs:
   deploy-cleanup:
     if: always()
     needs: [deploy-operator, deploy-test-vllm, deploy-test-sglang, deploy-test-trtllm]
-    runs-on: prod-default-small-v2
+    runs-on: prod-deploy-tester-v1
     steps:
       - uses: actions/checkout@de0fac2e4500dabe0009e67214ff5f5447ce83dd  # v6.0.2
       - name: Teardown vCluster
         if: needs.deploy-operator.outputs.namespace != '' && needs.deploy-operator.outputs.vcluster_name != ''
         uses: ./.github/actions/teardown-dynamo-operator
```

actionlint (1.7.12)

[error] missing input "kubeconfig_base64" which is required by action "Teardown Dynamo Operator" defined at "./.github/actions/teardown-dynamo-operator". all required inputs are "kubeconfig_base64", "vcluster_name", "vcluster_namespace"

(action)